projects/admin/src/app/deposit/editor/editor.component.ts

```diff
@@ -245,6 +246,78 @@ export class EditorComponent implements OnInit {
       });
   }
 
+  /**
+   * Extract metadata from PDF and populate deposit.
+   * @param event DOM Event
+   */
+  extractPdfMetadata(event: Event) {
+    event.preventDefault();
```

Comment:
It would be great to find an other solution as the method do not know that it is used from a `<a>`element.

Reply:
Yes you're right. Instead of this :
```
(click)="extractPdfMetadata($event)"
```

I propose :
```
(click)="$event.preventDefault(); extractPdfMetadata()"
```
With this, the method will be ignorant of which element of DOM is clicked.